src/Sarif/Autogenerated/SuppressionStates.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// The state of a result relative to a baseline of a previous run.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.16.0.0")]
+    [Flags]
```

Comment:
Note: with the new FlagsEnumConverter I wrote, the only change you'd need to make to your jschema code is to add this [Flags] attribute to an enum as required. You could look for a plural name or (preferable to you, I'm sure) add a new code hint making this explicit. [Flags] is required to get the correct ToString() behavior that allows parsing multiple values OR'd together.